docs/UserGuide.md

```diff
@@ -196,5 +204,6 @@ Action | Format, Examples
 Action | Format, Examples
 --------|------------------
 **List** | `list_session`
-**Add** | `add_session n/STUDENT_NAME d/DATE t/TIME l/LENGTH_OF_SESSION s/SUBJECT f/FEE`<br><br> e.g., `add_session n/John Doe d/14-02-2021 t/1800 l/2 s/Biology f/40`
-**Delete** | `delete_session n/STUDENT_NAME i/INDEX`<br><br>e.g. `delete_session n/John Lee i/1`
+**Find** | `find_session KEYWORD`<br><br>e.g. `find_session John`
+**Add** | `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`<br><br> e.g., `add_session n/John Doe d/2021-01-01 t/1800 l/120 s/Biology f/80`
```

Comment:
also just curious what is k

Reply:
Yup, thanks for spotting that, it should be k.
@nowknowing it's the prefix for duration (can be seen in the CliSyntax class in the Parser package).
I chose "/k" because "/l" was already used for STUDY_LEVEL in add_student.
If y'all have a more intuitive prefix letter, do suggest!